source/usr/lib/mcvirt/argument_validator.py

```diff
@@ -0,0 +1,82 @@
+"""Argument validators."""
+
+# Copyright (c) 2016 - I.T. Dev Ltd
+#
+# This file is part of MCVirt.
+#
+# MCVirt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MCVirt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import re
+
+
+class ArgumentValidator(object):
+    """Provide methods to validate argument values"""
+
+    @staticmethod
+    def validate_hostname(hostname):
+        """Validate a hostname"""
+        # Check length
+        if len(hostname) > 255 or not len(hostname):
+            raise TypeError
```

Comment:
I could be wrong here - but since you are just raising ``TypeError`` with no message the user will not know what has gone wrong, i.e. the hostname is invalid? Same applies to ``validate_network_name``